Formula/dnsviz.rb

```diff
@@ -51,63 +51,77 @@ def install
 
   test do
     message_1 = <<~EOS
-      6heBoAABAA4AAAABAAACAAEAAAIAAQAAQEQAFAFqDHJvb3Qtc2VydmVycwNuZXQAAAACAAEAA
-      EBEAAQBZsAeAAACAAEAAEBEAAQBa8AeAAACAAEAAEBEAAQBZcAeAAACAAEAAEBEAAQBZMAeAA
-      ACAAEAAEBEAAQBacAeAAACAAEAAEBEAAQBaMAeAAACAAEAAEBEAAQBY8AeAAACAAEAAEBEAAQ
-      BZ8AeAAACAAEAAEBEAAQBYsAeAAACAAEAAEBEAAQBbMAeAAACAAEAAEBEAAQBYcAeAAACAAEA
-      AEBEAAQBbcAeAAAuAAEAAEBEARMAAggAAAfpAFgqllBYGWTAmXsAco8uPM6BjFcEV4KkRvruu
-      C/2W4UHu5GcXkZKs9SbBc4i7KHFM+oyr4IPBZiNUEtn9L7rKJxyRdas+y3+uFld4jy7r0nhsx
-      cQnsQ1KvpwQPGJfzZnHYqSLofyXx2v3zWp1TItBHA4719vgGbj0ZsbnsonyvZt6N+t8cF0rFK
-      s2eWWcSwSFRGOOvxw2shQsvxQhDw4hC1jxkiG94xhJCbhZcmJq3DEc27I25amDYik9NkYROgf
-      mIA9+UcsmJ0tFJ0u4OBS6JN9iMomPOXcPFz/ZF3wGdCV1Jcx3Kj3ytOX2XccjAMF4kvbwx2/U
-      +pEm1xBDebTTOXC8biZ6gHF3AVRFgAAKQIAAACAAAAA
+      K+yBoAABAA4AAAABAAACAAEAAAIAAQAAV/cAFAFiDHJvb3Qtc2VydmVycwNuZXQAAAACAAEAA
```

Comment:
Can you explain why these messages were changed ?

Reply:
Let's improve the test so it doesn't need “fixing” every update. Otherwise this is close to unmaintainable.

Reply:
Please see if this is any better.  `brew test dnsviz` works on my end.